### PR TITLE
docs: Improved readability of device twin documentation for docs.kubeedge.io

### PR DESCRIPTION
**Pull request by sujithsimon22**
**What type of PR is this?**

 /kind documentation

**What this PR does / why we need it**:

This PR improves the readability of device twin documentation for the purpose of neat display in  docs.kubeedge.io by splitting explanation blocks into logical points.

### CODE REVIEW

File: docs/modules/devicetwin.md [hunk 1]
```diff
@@ -77,32 +77,39 @@ The following are the action callbacks which can be performed by the membership
...
-                                                                    This message arrives at the  devicetwin controller, which further sends the message to membership module.  The membership module
-                                                                     gets the devices associated with the edge node from the cache (context) and sends the information to the communication module, 
-                                                                     it also handles errors that may arise while performing the  aforementioned process and sends the error to the communication module
-                                                                      instead of device details.The communication module sends the information to the  eventbus component which further publishes the result on the 
-                                                                     specified MQTT topic (get membership result topic). 
+**dealMembershipGet**:    dealMembershipGet() deal gets the information  about the devices associated with the
```
Comment (rohitsardesai83) [inline docs/modules/devicetwin.md]: dealMembershipGet() ~~deal~~ gets the information  about

File: docs/modules/devicetwin.md [hunk 2]
```diff
@@ -77,32 +77,39 @@ The following are the action callbacks which can be performed by the membership
...
+                                                                    particular edge node, from the cache. 
+- The eventbus first receives a message on its subscribed topic (membership-get topic).
+- This message arrives at the  devicetwin controller, which further sends the message to membership module.  
+- The membership module gets the devices associated with the edge node from the cache (context) and sends the information to the communication module, 
+ it also handles errors that may arise while performing the  aforementioned process and sends the error to the communication module
```
Comment (rohitsardesai83) [inline docs/modules/devicetwin.md]: it -> It

File: docs/modules/devicetwin.md [hunk 3]
```diff
@@ -77,32 +77,39 @@ The following are the action callbacks which can be performed by the membership
...
-                                                                     gets the devices associated with the edge node from the cache (context) and sends the information to the communication module, 
-                                                                     it also handles errors that may arise while performing the  aforementioned process and sends the error to the communication module
-                                                                      instead of device details.The communication module sends the information to the  eventbus component which further publishes the result on the 
-                                                                     specified MQTT topic (get membership result topic). 
+**dealMembershipGet**:    dealMembershipGet() deal gets the information  about the devices associated with the
+                                                                    particular edge node, from the cache. 
+- The eventbus first receives a message on its subscribed topic (membership-get topic).
```
Comment (rohitsardesai83) [inline docs/modules/devicetwin.md]: remove extra spaces in between lines.

File: docs/modules/devicetwin.md [hunk 4]
```diff
@@ -77,32 +77,39 @@ The following are the action callbacks which can be performed by the membership
...
+- This message arrives at the  devicetwin controller, which further sends the message to membership module.  
+- The membership module gets the devices associated with the edge node from the cache (context) and sends the information to the communication module,
```
Comment (rohitsardesai83) [inline docs/modules/devicetwin.md]: module, -> module. period at the end.

File: docs/modules/devicetwin.md (hunk 2 above)
Comment (sujithsimon22) [reply docs/modules/devicetwin.md]: Done, Thanks

File: docs/modules/devicetwin.md (hunk 1 above)
Comment (sujithsimon22) [reply docs/modules/devicetwin.md]: Done, thanks

File: docs/modules/devicetwin.md (hunk 4 above)
Comment (sujithsimon22) [reply docs/modules/devicetwin.md]: Done, thanks

File: docs/modules/devicetwin.md (hunk 3 above)
Comment (sujithsimon22) [reply docs/modules/devicetwin.md]: Done, thanks

**Comment by rohitsardesai83** [comment]
@sujithsimon22 , please resolve the conflicts.

**Comment by sujithsimon22** [comment]
@rohitsardesai83 conflicts have been resolved, thanks :)

**Pull request merged**